lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/stack/StackPresenter.java

```diff
@@ -541,6 +541,10 @@ private LayoutParams getComponentLayoutParams(ComponentOptions component) {
         return new Toolbar.LayoutParams(LayoutParams.WRAP_CONTENT, LayoutParams.WRAP_CONTENT, component.alignment == Alignment.Center ? Gravity.CENTER : Gravity.START);
     }
 
+    public boolean hardwareBackButtonShouldPop(ViewController viewController) {
```

Comment:
Typically in functions that return a boolean, the function should begin with the word `should`. Currently, this is sounds yoda speak/french 😅